CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.1.0] - 12/09/2017
+### Added
+- Initial gem without release
+
+## [0.2.0] - 04/21/2020
+### Added
+- Add jenkins pipeline
+- Initial release
+
+## [0.2.1] - 07/09/2020
+### Added
+- MonotonicTickCount#timer: yield start_tick to block
+
+[0.2.0]: https://github.com/Invoca/monotonic_tick_count/releases/tag/v0.2.0
```

Comment:
The other entries should be accounted for as well (for the ones that have their corresponding versions tagged in github)

```
[0.2.1]: https://github.com/Invoca/monotonic_tick_count/releases/tag/v0.2.0...v0.2.1
[0.2.0]: https://github.com/Invoca/monotonic_tick_count/releases/tag/v0.1.0...v0.2.0
[0.1.0]: https://github.com/Invoca/monotonic_tick_count/releases/tag/v0.1.0
```

Reply:
the gem wasn't realeased until 0.2.0 and 0.2.1 has not yet been released.

Reply:
I'm just going to remove 0.1.0 since it was never released